research/src/components/specimens.js

```diff
@@ -10,13 +10,46 @@ const Specimens = ({ component, concept }) => {
   return (
     <div
       style={{
-        padding: hasImages ? '1em' : '2em',
-        marginBottom: '2em',
+        display: 'flex',
+        flexWrap: 'wrap',
+        marginBottom: '8px',
         border: '1px solid #ccc',
       }}
     >
-      {images.map((src, i) => (
-        <Image key={i} src={src} title={src} />
+      {images.map((image, i) => (
+        <div
+          key={i}
+          style={{
+            display: 'flex',
+            flex: '0 0 auto',
+            flexDirection: 'column',
+            padding: hasImages ? '8px' : '16px',
+            textAlign: 'center',
+          }}
+        >
+          <div style={{ margin: 'auto', padding: '4px' }}>
+            <Image
+              src={image.image}
+              title={image.image}
+              alt={`An image of the ${concept} concept on a ${component} component in ${image.sourceName}.`}
```

Comment:
It might be good to add an alt data field also, and make this formatted string the fallback if that field doesn't exist.

Reply:
Agreed, let's do in follow up PRs.